terraform/context_validate_test.go

```diff
@@ -595,7 +595,7 @@ func TestContext2Validate_providerConfig_bad(t *testing.T) {
 	}
 }
 
-func TestContext2Validate_providerConfig_badEmpty(t *testing.T) {
+func TestContext2Validate_providerConfig_skippedEmpty(t *testing.T) {
 	m := testModule(t, "validate-bad-pc-empty")
```

Comment:
To match things up, it could be nice to rename this file to match "skipped" versus "bad"